packages/terra-abstract-modal/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Changelog
 
 Unreleased
 ----------
+### Fixed
```

Comment:
I ain't sure if this can technically be treated as a bug but comments are welcome whether this should prolly be under 'Changed'.